Doc/tools/check-epub.py

```diff
@@ -1,24 +1,30 @@
-import sys
 from pathlib import Path
 
+CPYTHON_ROOT = Path(
+    __file__,  # cpython/Doc/tools/check-epub.py
+    '..',  # cpython/Doc/tools
+    '..',  # cpython/Doc
+    '..',  # cpython
+).resolve()
+EPUBCHECK_PATH = CPYTHON_ROOT / 'Doc' / 'epubcheck.txt'
 
-def main() -> int:
-    wrong_directory_msg = "Must run this script from the repo root"
-    if not Path("Doc").exists() or not Path("Doc").is_dir():
-        raise RuntimeError(wrong_directory_msg)
-
-    with Path("Doc/epubcheck.txt").open(encoding="UTF-8") as f:
-        messages = [message.split(" - ") for message in f.read().splitlines()]
 
-    fatal_errors = [message for message in messages if message[0] == "FATAL"]
+def main() -> int:
+    lines = EPUBCHECK_PATH.read_text(encoding='utf-8').splitlines()
+    fatal_errors = [line for line in lines if line.startswith('FATAL')]
 
```

Comment:
```suggestion
    errors = [line for line in lines if line.startswith('ERROR')]

    if errors:
        print()
        print(f'Warning: epubcheck reported {len(errors)} error(s):')
        print()
        print('\n'.join(errors))

```
Could we start to display errors as warnings for informational purposes?

Reply:
We now `cat` the entire file, so all errors, warnings, info, etc are visible.